R/build-search-docs.R

```diff
@@ -113,6 +113,14 @@ build_search_index <- function(pkg) {
 
   purrr::compact(index)
 
+  # Make URLs absolute if possible
+  url <- pkg$meta$url %||% "/"
```

Comment:
Are you sure this shouldn't be `""`? The url you use in the test doesn't have a trailing slash. Maybe worth testing the case of no default url?

Reply:
You were right! I added a test.

In the test I was surprised I had to make sure the file for both the cases was not the same. I think it's because for other snapshot tests, one does not have to worry about overwriting another snapshot. Good to know. :sweat_smile: